frontend/src/container/GridGraphLayout/GraphLayout.tsx

```diff
@@ -35,6 +37,16 @@ function GraphLayout({
 		role,
 	);
 
+	const getReactElementNode = (
+		isVisible: boolean,
+		Component: (props: ComponentProps) => JSX.Element,
+	): React.ReactNode =>
+		isVisible ? (
+			<Component setLayout={setLayout} />
+		) : (
+			<Spinner height="20vh" tip="Loading..." />
```

Comment:
this is violating https://github.com/SigNoz/signoz/issues/2074

also we are not fetching anything do we need to show the loader when the widget is already loaded ?

Reply:
Resolved, in latest commit.